TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -422,11 +422,34 @@ public override bool Visit(DataDescription dataDescription)
         {
             DataDescriptionEntry dataDescriptionEntry = dataDescription.CodeElement;
 
-            var levelNumber = dataDescriptionEntry.LevelNumber;
             //Check if the DataDescription is an empty group
-            if (levelNumber != null && dataDescription.IsDataDescriptionGroup && dataDescription.ChildrenCount == 0)
+            if (dataDescriptionEntry.LevelNumber != null && dataDescription.IsDataDescriptionGroup && dataDescription.ChildrenCount == 0)
             {
-                DiagnosticUtils.AddError(dataDescription, "A group item cannot be empty.", dataDescriptionEntry);
+                //Get all sibling nodes
+                var siblingNodes = dataDescription.Parent.Children.ToList();
+                //Get current node index
+                var nodeIndex = siblingNodes.IndexOf(dataDescription);
+
+                //Node is followed by at least one other node
+                if (siblingNodes.Count > nodeIndex + 1)
+                {
+                    //Get immediately following node
+                    var nextSibling = siblingNodes.ToArray()[nodeIndex + 1];
```

Comment:
Same thing here, no need to create an array, simply use `dataDescription.Parent.Children[nodeIndex + 1]`

Reply:
Done